.travis.yml

```diff
@@ -1,18 +1,21 @@
 language: python
 sudo: false
-matrix:
-  include:
-    - python: 3.5
-      env: TOXENV=py35
-env:
-    - TOXENV=py27
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=pypy
-    - TOXENV=pypy3
+python:
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy-5.4.1
```

Comment:
Can we add 3.6 while we're at it?  Or are tests currently failing on 3.6?  Or do you just wish to keep the concerns separate and think 3.6 support belongs in a separate PR?